data/phone/text/anthony_caller.asm

```diff
@@ -20,7 +20,7 @@ AnthonyAteBerriesText:
 	line "was delicious!"
 	done
 
-UnknownText_0x176aef:
+AnthonyLotsaWildText:
```

Comment:
Try to use direct excerpts from the text for labels, or if you do paraphrase (as with `AnthonyAteBerriesText`) avoid slang. (This also happens to kind of clash with `AnthonyLostAMonText` right below it; I thought it was a typo.)